docs/appendices/release-notes/5.5.2.rst

```diff
@@ -79,6 +79,17 @@ Packaging Changes
   also contains packages for ARM64.
 
 
+Security Fixes
```

Comment:
Should this also go into the release notes for the other versions, or will you do that in the backports?

Reply:
Planned to do this in the backports and afterwards bring the new release to master. Or do you prefer the other way around?

Reply:
> Or do you prefer the other way around?

No preference, both okay

> Should we move this at the most top? (Before the "packaging" section)

Sounds good, yes